apps/condo/domains/property/components/AddressSuggestionsSearchInput.tsx

```diff
@@ -1,13 +1,15 @@
 import { grey } from '@ant-design/colors'
```

Comment:
How about iteration of condo-ui refactor and change colors?

Reply:
Removed this import because the actual color is set in the `TextHighlighter` component.